cmd/init-repo/main.go

```diff
@@ -38,6 +38,8 @@ Command line flags override config values.
 
 func main() {
 	configFilePath := flag.String("config", "", "the config file in yaml format")
+	githubHost := flag.String("github-host", "", "the address of github (eg. github.com)")
+	basePackage := flag.String("base-package", "", "the name of the package base (eg. k8s.io)")
```

Comment:
"defaults to k8s.io if source-repo is kubernetes and <github-host>/<target-org> otherwise"

Reply:
fixed.